packages/mdc-dialog/README.md

```diff
@@ -414,7 +414,7 @@ a focus trapping solution for your component code.**
 
 ```js
 const {activate, deactivate} =
-  util.createFocusTrapInstance(surfaceEl, initialFocusEl, focusTrapFactory = require('focus-trap'));
+  util.createFocusTrapInstance(surfaceEl, focusTrapFactory = require('focus-trap'), initialFocusEl);
```

Comment:
Looking at this line again... what is it trying to accomplish in terms of documentation? If this is an invocation, `focusTrapFactory =` doesn't make sense. If this is just trying to reflect the API, then I guess it should also have `initialFocusEl = null`.

Reply:
* Removed the ` = require('focus-trap')` part
* Clarified that `initialFocusEl` defaults to the first focusable element